src/__tests__/components/login.test.tsx

```diff
@@ -0,0 +1,32 @@
+import * as React from "react";
+import * as renderer from "react-test-renderer";
+import configureStore from "redux-mock-store";
+
+import { configure, mount, shallow } from "enzyme";
+import * as Adapter from "enzyme-adapter-react-16";
+import { Provider } from "react-redux";
+
+import Login from "../../components/login/login";
+import User from "../../models/user";
+
+jest.mock("history");
+
+describe("Login component", () => {
+    it("renders correctly", () => {
+        const initialState = {
+            authentication: { currentUser: new User("public", "private"), isLoading: false },
+        };
+
+        const mockStore = configureStore();
+
+        configure({ adapter: new Adapter() });
+        const store = mockStore(initialState);
+        const container = shallow(
+            <Provider store={store}>
+                <Login />
+            </Provider>,
+        ).dive();
+        expect(Object.keys(container.props()).length).toEqual(1);
+        // TODO: Test if state is set correctly
```

Comment:
Resolve this TODO.

Reply:
I saw in that in the develop branch the __tests__ folder doesn't exist anymore, so I deleted the whole folder here also.